react-with-graphql-tutorial.md

```diff
@@ -720,19 +721,19 @@ class App extends Component {
 }
 {{< /highlight >}}
 
-The previous setup for the form, input field(s), a submit button, `onChange()` and `onSubmit()` class methods is a common way to implement forms in React. The only addition is the initial data fetching in the `componentDidMount()` lifecycle method which only helps to improve the user experience by providing an initial state from a backend. In summary, it is a neat boilerplate to [fetch data in React from a third-party API](https://www.robinwieruch.de/react-fetching-data/).
+The previous setup for the form--using input field(s), a submit button, `onChange()` and `onSubmit()` class methods--is a common way to implement forms in React. The only addition is the initial data fetching in the `componentDidMount()` lifecycle method to improve user experience by providing an initial state from the backend. It is a useful foundation for [fetching data from a third-party API in React](https://www.robinwieruch.de/react-fetching-data/).
```

Comment:
"... lifecycle method to improve user experience by providing an initial state from the backend." to "... lifecycle method to improve user experience by providing an initial state for the query to request data from the backend."